ACPCampaignClassic.podspec

```diff
@@ -1,27 +1,60 @@
 Pod::Spec.new do |s|
   s.name         = "ACPCampaignClassic"
-  s.version      = "1.1.0"
-  s.summary      = "Campaign Classic framework for Adobe Experience Platform SDK. Written and Supported by Adobe."
+  s.version      = "2.0.0"
+  s.summary      = "Campaign Classic framework for Adobe Experience Cloud SDK. Written and maintained by Adobe."
   s.description  = <<-DESC
-                   The campaign classic framework provides APIs that allow use of the Campaign Classic product in the V5 Adobe Experience Platform SDK.
+                   The campaign classic framework provides APIs that allow use of the Campaign Classic product in the V5 Adobe Experience Cloud SDK.
```

Comment:
'V5' on this line should be removed, and replace use of 'framework' with 'library'.